components/form/FormItem.razor.cs

```diff
@@ -223,15 +254,54 @@ void IFormItem.AddControl<TValue>(AntInputComponentBase<TValue> control)
                 builder.CloseComponent();
             };
 
-            if (control.ValueExpression is not null)
+			if (control.ValueExpression is not null)
                 _propertyReflector = PropertyReflector.Create(control.ValueExpression);
             else
                 _propertyReflector = PropertyReflector.Create(control.ValuesExpression);
 
-            if (_propertyReflector.RequiredAttribute != null)
+        }
+
+        ValidationResult[] IFormItem.ValidateField()
+        {
+            if (Rules == null)
             {
-                _labelCls = $"{_prefixCls}-required";
+                return Array.Empty<ValidationResult>();
             }
+
+            var results = new List<ValidationResult>();
+
+            var propertyInfo = _fieldIdentifier.Model.GetType().GetProperty(_fieldIdentifier.FieldName);
```

Comment:
A micro-optimization, but maybe this `propertyInfo` should be a private field? Then initialize it only once, if used. You are going to avoid reflection in here on every validation.